os/drivers/Kconfig

```diff
@@ -322,6 +322,7 @@ endif # RTC
 
 menuconfig WATCHDOG
 	bool "Watchdog Timer Support"
+	select S5J_WATCHDOG
```

Comment:
Dont make a dependancy with specific hw config in driver

Reply:
fixed.

Reply:
I fixed.